bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,246 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+import json
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdin=True,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json')
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         remove_empty_rulesets: bool = True,
+                         force_semicolon: bool = True,
+                         color_case: str = "upper",
```

Comment:
what about changing this to a boolean i dunno maybe ``upper_case_color: bool=True``
this is just a preference anyway :)